src/ui/layout/search/GrandSearch.vue

```diff
@@ -127,12 +132,17 @@ export default {
                 this.searchLoading = false;
                 this.showSearchResults();
             } catch (error) {
-                console.error(`😞 Error searching`, error);
                 this.searchLoading = false;
 
                 if (this.abortSearchController) {
                     delete this.abortSearchController;
                 }
+
+                // Is this coming from the AbortController?
+                // If so, we can swallow the error. If not, 🤮 it to console
+                if (!error.name === 'AbortError') {
```

Comment:
Is the `!error.name` intentional or a typo ?

Reply:
typo! nice catch